client/src/components/Warning/index.ts

```diff
@@ -0,0 +1 @@
+export { Warning } from './Warning';
```

Comment:
Minor one: if you plan to have only one file with code file in this folder, then you can put it directly in `index.tsx` and remove `index.ts`

Reply:
fixed